content/en/docs/03/03_user-defined-rules.md

```diff
@@ -80,17 +80,8 @@ Let's take a look at the main components that can be configured applying a custo
 * **for:** Duration of the fulfilled condition until the alarm is triggered. If not set, alerts are triggered at their first occurrence.
 * **labels:** Metadata that can be used to dynamically route the alert to a corresponding receiver.
 
-Verify the rule by accessing the Thanos ruler web interface.
-You can find out its URL by looking at the `thanos-ruler` route in the `openshift-user-workload-monitoring`.
-
-{{% details title="Hints" mode-switcher="normalexpertmode" %}}
-
-```bash
-oc -n openshift-user-workload-monitoring get route thanos-ruler -o go-template='https://{{ .spec.host }}/alerts{{ "\n" }}'
-```
-
-{{% /details %}}
-
+Note: Starting with OpenShift 4.11, none of the monitoring components (Thanos, Prometheus...) are exposed anymore.
+This means that, despite routes still existing in the cluster, no corresponding Web UI is accessible.
```

Comment:
Same here, I'd leave that out. There'll be a version that will completely be rid of the routes and our description will be out-of-date again.

Reply:
Roger, I've removed the notice altogether as per commit f25dd8133b06ebc0ba6f2006310d3df7eec3c6a4.  
This also applies to the prior comment.